Marlin/Configuration.h

```diff
@@ -1503,6 +1503,37 @@
 //
 //#define OLED_PANEL_TINYBOY2
 
+//
+// Makeboard 3D Printer Parts 3D Printer Mini Display 1602 Mini Controller
+// https://www.aliexpress.com/item/Micromake-Makeboard-3D-Printer-Parts-3D-Printer-Mini-Display-1602-Mini-Controller-Compatible-with-Ramps-1/32765887917.html
+//
+// 2 line display.  
+//
+//#define MAKEBOARD_MINI_2_LINE_DISPLAY_1602
+
+//
+// MKS MINI12864 with graphic controller and SD support 
+// http://reprap.org/wiki/MKS_MINI_12864
+//
+//
+//#define MKS_MINI_12864
+
+//
+// Factory display for Creality CR-10 
+// https://www.aliexpress.com/item/Universal-LCD-12864-3D-Printer-Display-Screen-With-Encoder-For-CR-10-CR-7-Model/32833148327.html
+//
+// this is a a RAMPS-compaitible version, but with a single 10-pin connector instead of 2, for CR-10 owners who want to replace the Melzi Creality board, but retain the display
+// 
+//#define CR10_STOCKDISPLAY
```

Comment:
Let's see if I understand correctly. This is the option someone would select using the CR-10 display on a RAMPS-compatible board, and since the CR-10 sees it as a standard Melzi plus RRD Full Graphical Smart Display, this option may also be selected for this display in a CR-10 — since it just forces `REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER` to be enabled.

I think I have enough to work with from this PR to clean it up and merge it. I will follow up with a new PR soon.

Reply:
yes